src/mscorlib/src/System/IO/DriveNotFoundException.cs

```diff
@@ -18,7 +18,7 @@ namespace System.IO {
     //Thrown when trying to access a drive that is not availabe.
     [Serializable]
     [System.Runtime.InteropServices.ComVisible(true)]
-    public class DriveNotFoundException : IOException {
+    internal class DriveNotFoundException : IOException {
```

Comment:
Does _Error actually need it, though?  With DriveInfo being in CoreFx, I'm wondering if anything in CoreCLR would actually generate the error code that would trigger that exception to be created/thrown.